packages/forms-web-app/src/controllers/lpa-dashboard/appeal-details.js

```diff
@@ -1,40 +1,37 @@
-const { getAppealByLPACodeAndId } = require('../../lib/appeals-api-wrapper');
+const {
+	getAppealByLPACodeAndId,
+	getAppealDocumentMetaData
+} = require('../../lib/appeals-api-wrapper');
 const {
 	VIEW: {
 		LPA_DASHBOARD: { DASHBOARD, APPEAL_DETAILS }
 	}
 } = require('../../lib/views');
-const {
-	constants: { APPEAL_ID }
-} = require('@pins/business-rules');
 const dateFns = require('date-fns');
 
-const calculateQuestionnaireDueDate = (appeal) => {
-	const { appealValidDate } = appeal;
-	let questionnaireDueDate = {};
-	switch (appeal.type) {
-		case APPEAL_ID.HOUSEHOLDER:
-			questionnaireDueDate = dateFns.addBusinessDays(appealValidDate, 5);
-			break;
-		default:
-			throw new Error('Unsupported appeal type');
-	}
-	console.log(questionnaireDueDate);
-	return questionnaireDueDate;
-};
-
 const getAppealDetails = async (req, res) => {
-	const { lpaCode, id } = req.params;
-	const appeal = await getAppealByLPACodeAndId(lpaCode, id);
+	const { id } = req.params;
+	const { lpaUser } = req.session;
+	const caseReference = encodeURIComponent(id);
+	const appeal = await getAppealByLPACodeAndId(lpaUser.lpaCode, caseReference);
+	const applicationFormMetaData = await getAppealDocumentMetaData(
+		caseReference,
+		'Planning%20application%20form'
+	);
+	const applicationForm = {
+		filename: applicationFormMetaData.filename,
+		documentURI: applicationFormMetaData.documentURI
+	};
 
 	return res.render(APPEAL_DETAILS, {
 		dashboardLink: `/${DASHBOARD}`,
 		appeal,
+		applicationForm,
 		questionnaireDueDate: (() => {
 			return new Intl.DateTimeFormat('en-GB', {
 				dateStyle: 'full',
 				timeZone: 'Europe/London'
-			}).format(calculateQuestionnaireDueDate(appeal));
+			}).format(dateFns.parseISO(appeal.questionnaireAndCaseFileDue));
```

Comment:
Just a minor tweak to use this field please questionnaireDueDate. I think the reasoning is that front office primarily handles questionnaires from LPA officers, I think the case file might be a back office concern